### PR TITLE
Add storybook render tests with CI integration

### PR DESCRIPTION
**Pull request by EtDu**
- Start of render tests for storybook components, they simply fail if the component does not render
- Create a separate jest config `jest.stories.config.js`
- New command `yarn storybook:test`
- Attach command to CI workflow

### CODE REVIEW

File: jest.stories.config.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      statements: 43,
+    },
+  },
+  // TODO: enable resetMocks
+  // resetMocks: true,
+  restoreMocks: true,
+  setupFiles: ['<rootDir>/test/setup.js', '<rootDir>/test/env.js'],
+  setupFilesAfterEnv: ['<rootDir>/test/jest/setup.js'],
+  testMatch: ['<rootDir>/ui/**/*stories.test.js'],
+  testTimeout: 2500,
```
Comment (georgewrmarshall) [inline jest.stories.config.js:21]:
```suggestion
  testTimeout: 2500,
    transform: {
    '^.+\\.[tj]sx?$': 'babel-jest',
    '^.+\\.mdx$': '@storybook/addon-docs/jest-transform-mdx',
  },
```

**Review by georgewrmarshall — COMMENTED** [review]
Got it working by adding the suggestions below. I'm still not sure if writing unit tests for every story is going to be maintainable though? Note this message on Storybook website under [unit testing](https://storybook.js.org/docs/react/workflows/unit-testing/) which links to visual regression testing suggestions like storybook shots or chromatic.

Screen Shot 2021-11-01 at 11 28 26 AM

File: ui/components/app/transaction-total-banner/transaction-total-banner.stories.js [hunk 2]
```diff
@@ -19,3 +19,5 @@ export const basic = () => {
     />
   );
 };
+
+DefaultStory.storyName = 'Default';
```
Comment (georgewrmarshall) [inline ui/components/app/transaction-total-banner/transaction-total-banner.stories.js:23]: Needs newline
```suggestion
DefaultStory.storyName = 'Default';

```

**Review by georgewrmarshall — DISMISSED** [review]
lgtm

**Review by georgewrmarshall — APPROVED** [review]
lgtm

**Review by darkwing — APPROVED** [review]

**Review by jj-crypto — APPROVED** [review]

**Pull request merged**